db/seeds.rb

```diff
@@ -2,23 +2,27 @@
 
 PaperTrail.enabled = false
 
-seed_path = %w[db seeds]
-load Rails.root.join(*seed_path, "initial_seed.rb").to_s
-load Rails.root.join(*seed_path, "schedules.rb").to_s
+if ENV["LEGACY_SEEDS"] == "true"
+  seed_path = %w[db legacy_seeds]
+  load Rails.root.join(*seed_path, "initial_seed.rb").to_s
+  load Rails.root.join(*seed_path, "schedules.rb").to_s
 
-if %w[development deployed_development test sandbox].include?(Rails.env)
-  %w[test_data dummy_structures appropriate_bodies].each do |seed|
-    load Rails.root.join(*seed_path, "#{seed}.rb").to_s
+  if %w[development deployed_development test sandbox].include?(Rails.env)
+    %w[test_data dummy_structures appropriate_bodies].each do |seed|
+      load Rails.root.join(*seed_path, "#{seed}.rb").to_s
+    end
   end
-end
 
-if Rails.env.development? || Rails.env.deployed_development?
-  FeatureFlag::PERMANENT_SETTINGS.each do |feature|
-    FeatureFlag.activate(feature)
-  end
-  FeatureFlag::TEMPORARY_FEATURE_FLAGS.each do |feature|
-    FeatureFlag.activate(feature)
+  if Rails.env.development? || Rails.env.deployed_development?
+    FeatureFlag::PERMANENT_SETTINGS.each do |feature|
+      FeatureFlag.activate(feature)
+    end
+    FeatureFlag::TEMPORARY_FEATURE_FLAGS.each do |feature|
+      FeatureFlag.activate(feature)
+    end
   end
+else
+  load(Rails.root.join(*%w[db new_seeds run.rb]).to_s)
```

Comment:
oops missed this, just a quick q do we need to worry about envs before we run this as in the legacy one we are explicitly setting those? also do we want to enable the feature flags as well (as those aren't included in the new seeds run file)